lib/commands/location.js

```diff
@@ -66,5 +67,26 @@ commands.setGeoLocation = async function setGeoLocation (location) {
   }
 };
 
+/**
+ * Reset the location service on real device.
+ * Raises not implemented error for simulator.
+ * @throws {Error} If the device is simulator, or 'resetLocation' raises an error.
+ */
+commands.mobileResetLocationService = async function mobileResetLocationService () {
+  if (this.isSimulator()) {
+    throw new errors.NotImplementedError();
+  }
+
+  const service = await services.startSimulateLocationService(this.opts.udid);
+  try {
+    service.resetLocation();
```

Comment:
should it be awaited?

Reply:
I think no -> https://github.com/appium/appium-ios-device/blob/b47d5b394f884b597281ebabd48144e6fc17d23f/lib/simulatelocation/index.js#L13
service.setLocation in the above `setGeoLocation` also did not, so I did this.